src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private String color;
+    private double radius;
+
+    public Circle(String color, double radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Square: color="
```

Comment:
The message in the `draw` method says 'Square' but this is the 'Circle' class. Please correct the text to match the class it refers to.